src/Illuminate/Cookie/CookieJar.php

```diff
@@ -30,6 +30,13 @@ class CookieJar implements JarContract
     protected $secure = false;
 
     /**
+     * The default httpOnly setting (defaults to true).
+     *
+     * @var bool
+     */
+    protected $httpOnly = true;
```

Comment:
>  and it can be override by passing another value to make function.

When anyone upgrade from say `v5.2.24` to `v5.2.25` (if this get accepted). People shouldn't be worry about any configuration change and everything should works as it was before.